kubernetes/loculus/values_e2e_and_dev.yaml

```diff
@@ -7,5 +7,6 @@ secrets:
       externalMetadataUpdaterPassword: "external_metadata_updater"
       backendUserPassword: "backend"
 createTestAccounts: true
+disableEnaSubmission: false
```

Comment:
maybe we can disable this here as well, enabling submission for the e2e tests seems wrong, for local testing it can be changed manually. 
```suggestion

```

Reply:
Sorry I forgot about this PR then almost created a new one but found the branch name was taken and found it again!